src/main/java/net/sf/jabref/sql/DatabaseUtil.java

```diff
@@ -9,9 +9,9 @@
 
 import net.sf.jabref.MetaData;
 
-public class DBImporterExporter {
+public class DatabaseUtil {
```

Comment:
I don't like these static Util classes... couldn't you find a better home for these methods?